### DISCUSSION TITLE
New Feature: Multiple intensity preferences available

### DISCUSSION THREAD

**Discussion by rosarian** [OpenRCT2/OpenRCT2#17032] [Feature requests]
Currently the peep intensity preference is set globally for a scenario (Gentle Glen vs. Adrenaline Heights vs. most others).
It would be interesting for this to be tied instead to an individual peep spawn point.  This way one side of a park could be the Kiddie section, the opposite side for the insane and reckless, and the middle for in between.  It might also help reduce peeps getting lost.

**Comment by IntelOrca** [comment]
I don't think I see the real life resemblance of this. Wouldn't it be more logical (if AI / pathfinding was improved) for the guests to use a transportation ride to access the area of the park they prefer?

**Reply by Broxzier** [reply]
Toverland in the Netherlands has two entrances, one for a kids area and one for the theme park. Once inside you can freely go from one area to the other.

I think this would be best implemented as a plugin. When a peep is spawned, it checks which entrance it goes to, then sets the intensity for the guest.

**Comment by rosarian** [comment]
It just seems to me that when the number of spawn points was increased from 1 to many that the intensity preference should no longer be global.  This would simulate different neighborhoods - some with lots of children, some with wacky adolescents, etc.
I also would have thought that increasing the number of park entrances from 2 to many would make sense to de-globalize the park entrance style and put it with the entrance instead.  If these things are done through a plugin they would not be saved with the park or scenario.
